Documentation/project-docs/clr-configuration-knobs.md

```diff
@@ -1,829 +1,1020 @@
+`System.GC.RetainVM` | Put segments that should be deleted on a standby list for future use instead of releasing them back to the OS | boolean
+`System.Threading.ThreadPool.MinThreads` | Override MinThreads for the ThreadPool worker pool | numeric
+`System.Threading.ThreadPool.MaxThreads` | Override MaxThreads for the ThreadPool worker pool | numeric
+
+
 
 ## Environment/Registry Configuration Knobs
 
-This table is machine-generated from commit 456b418 on 08/22/17. It might be out of date.
+This table was machine-generated from commit [5f0c8040](https://github.com/dotnet/coreclr/commit/5f0c80406a370c5d6b41a379e015511cdaf07dda) on 2018-01-15. It might be out of date.
 
 When using these configurations from environment variables, the variables need to have the `COMPlus_` prefix in their names. e.g. To set DumpJittedMethods to 1, add the environment variable `COMPlus_DumpJittedMethods=1`.
 
 See also [Setting configuration variables](../building/viewing-jit-dumps.md#setting-configuration-variables) for more information.
 
+#### Tables
+1. [AppDomain Configuration Knobs](#appdomain-configuration-knobs)
+2. [ARM Configuration Knobs](#arm-configuration-knobs)
+3. [Assembly Loader Configuration Knobs](#assembly-loader-configuration-knobs)
+4. [Conditional breakpoints Configuration Knobs](#conditional-breakpoints-configuration-knobs)
+5. [CSE Configuration Knobs](#cse-configuration-knobs)
+6. [Debugger Configuration Knobs](#debugger-configuration-knobs)
+7. [Diagnostics (internal general-purpose) Configuration Knobs](#diagnostics-internal-general-purpose-configuration-knobs)
+8. [Exception Handling Configuration Knobs](#exception-handling-configuration-knobs)
+9. [Garbage collector Configuration Knobs](#garbage-collector-configuration-knobs)
+10. [GDBJIT Configuration Knobs](#gdbjit-configuration-knobs)
+11. [IBC Configuration Knobs](#ibc-configuration-knobs)
+12. [Interop Configuration Knobs](#interop-configuration-knobs)
+13. [Interpreter Configuration Knobs](#interpreter-configuration-knobs)
+14. [JIT Configuration Knobs](#jit-configuration-knobs)
+15. [JIT Hardware Intrinsics Configuration Knobs](#jit-hardware-intrinsics-configuration-knobs)
+16. [Jit Pitching Configuration Knobs](#jit-pitching-configuration-knobs)
+17. [JIT64 Configuration Knobs](#jit64-configuration-knobs)
+18. [Loader Configuration Knobs](#loader-configuration-knobs)
+19. [Loader heap Configuration Knobs](#loader-heap-configuration-knobs)
+20. [Log Configuration Knobs](#log-configuration-knobs)
+21. [MDA ? Configuration Knobs](#mda--configuration-knobs)
+22. [MetaData Configuration Knobs](#metadata-configuration-knobs)
+23. [Native Binder Configuration Knobs](#native-binder-configuration-knobs)
+24. [NGEN Configuration Knobs](#ngen-configuration-knobs)
+25. [Perf Configuration Knobs](#perf-configuration-knobs)
+26. [Perfcounters Configuration Knobs](#perfcounters-configuration-knobs)
+27. [Performance Tracing Configuration Knobs](#performance-tracing-configuration-knobs)
```

Comment:
Perhaps `Perf` and `Perfcounters` could be renamed to Performance and Performance Counters respectively?

Reply:
`Perfcounters` knobs are dead code in CoreCLR. The one knob in this category can be deleted.